.github/workflows/ci.yml

```diff
@@ -52,6 +52,26 @@ jobs:
       - name: Lint code for consistent style
         run: bin/rubocop -f github
 
+  lint_js:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20"
+          cache: "npm"
+
+      - name: Install dependencies
+        run: npm install
+        shell: bash
+
+      - name: Lint/Format js code
+        run: npm run style:check
```

Comment:
To keep this as low-overhead as possible, I think we should just enforce the linting:

```
run: npm run lint
```

The formatting is a nice addition that I'll be using locally, but don't want that to be a requirement for new contributors

Reply:
changed it, even if its not a best practice.

Reply:
@oxdev03 correct, not best practice and that's 100% fine at our stage/size